rcsb/workflow/cli/BcifExec.py

```diff
@@ -0,0 +1,149 @@
+##
+# File:    BcifExec.py
+# Author:  James Smith
+# Date:    21-Feb-2025
+##
+
+"""
+Entry point for bcif command line interface.
+"""
+
+__docformat__ = "google en"
+__author__ = "James Smith"
+__email__ = "james.smith@rcsb.org"
+__license__ = "Apache 2.0"
+
+import argparse
+import logging
+import time
+from rcsb.workflow.wuw.BcifWorkflow import BcifWorkflow
+
+logger = logging.getLogger("bcif")
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler()
+formatter = logging.Formatter(
+    fmt="%(asctime)s @%(process)s [%(levelname)s]-%(module)s: %(message)s"
+)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
```

Comment:
Can you move this logging setup to be within the `main()` function, following the argument parsing stuff? This would allow you to also add flags for `--debug` and `--log_file_path`, and would only be executed when the CLI is called, which is I think what we want to do. 

You could follow this as an example: https://github.com/rcsb/py-rcsb_db/blob/master/rcsb/db/cli/RepoLoadExec.py#L180-L199

And note how the `logging.getLogger()` is done at the top of the file: https://github.com/rcsb/py-rcsb_db/blob/master/rcsb/db/cli/RepoLoadExec.py#L48